SU2_CFD/include/numerics/NEMO/CNEMONumerics.hpp

```diff
@@ -32,6 +32,8 @@
 #include "../../fluid/CMutationTCLib.hpp"
 #include "../../fluid/CSU2TCLib.hpp"
 #include "../../../../Common/include/toolboxes/geometry_toolbox.hpp"
+#include "../../../Common/include/geometry/CGeometry.hpp"
+#include "../../variables/CNEMOEulerVariable.hpp"
```

Comment:
This will not be relevant after you fix the sensor, but avoid including derived classes in base classes or other families, the one good thing about polymorphism is isolation which allows reducing compilation dependencies.
Here you would forward declare CNEMOEulerVariable instead of including, and then include only in the cpp where it is needed (aumsplusm in this case).
```suggestion
```

Reply:
Removed as I am using sensors now